app/models/runner.js

```diff
@@ -137,12 +151,16 @@ async function writeProcessorFile(fileContent) {
     }
 }
 
-function injectPlugins(testFile, jobConfig) {
-    const metricsPluginName = jobConfig.metricsPluginName.toLowerCase();
-    const metricsAdapter = require(`../adapters/${metricsPluginName}Adapter`);
-    let asciiMetricsExportConfig = (Buffer.from(jobConfig.metricsExportConfig, 'base64').toString('ascii'));
-    let parsedMetricsConfig = JSON.parse(asciiMetricsExportConfig);
-    testFile.config.plugins = metricsAdapter.buildMetricsPlugin(parsedMetricsConfig, jobConfig);
+async function getProcessorPathIfExists(jobConfig, test) {
```

Comment:
i would simplify it by
1. when you download the 'file' you know it base 64 so just decode it
2. get rid of writeProcessorFile func, it seems like it's not really needed.

Reply:
done